### ISSUE TITLE
Do not redirect if current site is the redirected one

### ISSUE THREAD

**Issue by IceDBorn** [einaregilsson/Redirector#283]
I'm using [Piped](piped.kavin.rocks) that has a Watch on Youtube button, I would like to not redirect back to piped after pressing the button. An option for excluded urls would be nice to have.

**Comment by IceDBorn** [comment]
![image](https://user-images.githubusercontent.com/51162078/137643084-45d89884-3772-45be-9b3f-90ec3a7bc2cb.png)
What pattern do I use though?

**Comment by einaregilsson** [comment]
Well, if you use the watch on YouTube button, what url does it send you to? Just a normal youtube watch url, or does it have any parameters, any special things that could be used to tell it apart from when you should be redirected to piped?

**Comment by IceDBorn** [comment]
Yeah, that's a possible solution, but adding the feature I requested would be better. You can close the issue now. Thanks for your help!
